packages/react-magma-dom/src/components/NavTabs/NavTab.tsx

```diff
@@ -182,12 +196,14 @@ export const NavTab = React.forwardRef<any, NavTabProps>((props, ref) => {
           isInverse={isInverse}
           isFullWidth={isFullWidth}
           orientation={orientation}
+          ref={styledTabRef}
           theme={theme}
         />
       ) : (
         <StyledTab
           {...other}
-          ref={ref}
```

Comment:
Does this ref need to be removed? I just want to make sure we aren't breaking any functionality that currently exists in case someone is using it.

Reply:
That might be an older pull, it should be showing `ref={styledTabRef}`.

Reply:
Looks like `ref={ref}` was only on `StyledTab` but you've added `ref={styledTabRef}` to `StyledCustomTab`
![image](https://user-images.githubusercontent.com/91160746/201762892-a4e162eb-5af3-4ae0-8d08-6213a55664ff.png)

Reply:
In case the user wants to have the same focus with an icon based NavTab as well.